doc/rest-api.md

```diff
@@ -2933,42 +3202,48 @@ Return:
 
 Input (bootstrap a new cluster):
 
-    {
-        "server_name": "lxd1",
-        "enabled": true,
-    }
+```json
+{
+    "server_name": "lxd1",
+    "enabled": true,
+}
+```
 
 Return background operation or standard error.
 
 Input (request to join an existing cluster):
 
-    {
-        "server_name": "node2",
-        "server_address": "10.1.1.102:8443",
-        "enabled": true,
-        "cluster_address": "10.1.1.101:8443",
-        "cluster_certificate": "-----BEGIN CERTIFICATE-----MIFf\n-----END CERTIFICATE-----",
-        "cluster_password": "sekret",
-        "member_config": [
-            {
-                "entity": "storage-pool",
-                "name": "local",
-                "key": "source",
-                "value": "/dev/sdb",
-            },
-            {
-                "entity": "network",
-                "name": "lxdbr0",
-                "key": "bridge.external_interfaces",
-                "value": "vlan0",
-            },
-    }
+```json
+{
+    "server_name": "node2",
+    "server_address": "10.1.1.102:8443",
+    "enabled": true,
+    "cluster_address": "10.1.1.101:8443",
+    "cluster_certificate": "-----BEGIN CERTIFICATE-----MIFf\n-----END CERTIFICATE-----",
+    "cluster_password": "sekret",
+    "member_config": [
+        {
+            "entity": "storage-pool",
+            "name": "local",
+            "key": "source",
+            "value": "/dev/sdb",
+        },
+        {
+            "entity": "network",
+            "name": "lxdbr0",
+            "key": "bridge.external_interfaces",
+            "value": "vlan0",
+        },
+}
+```
```

Comment:
Not sure what's going on around here but github looks rather unhappy.

Reply:
It due to the upper comma `},`

Reply:
oh, missing closing bracket `]`